### PR TITLE
Relnotes 5.1.3

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -3,17 +3,78 @@
...
+| https://issues.couchbase.com/browse/MB-31838[MB-31838^]
+| When data corruption is detected on a data service node, relevant information that can help troubleshoot is now being logged.
+
+| https://issues.couchbase.com/browse/MB-31714[MB-31714^]
+| When bucket memory usage exceeded the high watermark and reached 99% of the bucket quota, a race condition in KV-Engine DCP consumer was exposed, resulting in DCP messages being processed out-of-order.
+
+| https://issues.couchbase.com/browse/MB-31569[MB-31569^]
+| Fixed an issue where appending to a document reset the expiration value to 0, disabling TTL expiry.
+
+| https://issues.couchbase.com/browse/MB-31569[MB-31548^]
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc]: Wrong link

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -3,17 +3,78 @@
...
+[#table_fixedissues_index_513,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31746[MB-31746^]
+| Document mutations with numbers greater than MaxInt64 encountered an error and were not indexed instead of being indexed with loss of precision.
+|===
+
+==== Tools
+[#table_fixedissues_install_513,cols="25,66"]
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc]: Should be #table_fixedissues_tools_513

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -3,17 +3,78 @@
...
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31567[MB-31567^]
+| Improve XDCR connection management to prevent XDCR replication getting stuck while creating a checkpoint.
+|===
+
+==== Data Service
+
+[#table_fixedissues_index_513,cols="25,66"]
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc]: Should be #table_fixedissues_data_513

**Review by eric-schneider — CHANGES_REQUESTED** [review]

File: modules/release-notes/pages/relnotes.adoc [hunk 4]
```diff
@@ -3,17 +3,78 @@
...
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31746[MB-31746^]
+| Document mutations with numbers greater than MaxInt64 encountered an error and were not indexed instead of being indexed with loss of precision.
+|===
+
+==== Tools
```
Comment (eric-schneider) [inline modules/release-notes/pages/relnotes.adoc:58]: Add space

Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc:58]: fixed in update

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc]: fixed in update

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc]: fixed in update

**Review by eric-schneider — APPROVED** [review]

**Pull request merged**